ngrok/README.md

```diff
@@ -18,16 +18,22 @@
 [apache-url]: https://github.com/ngrok/ngrok-rs/blob/main/LICENSE-APACHE
 
 [Website](https://ngrok.com) |
-[API Docs (unstable)](https://ngrok.github.io/ngrok-rs/ngrok)
-
-ngrok is a globally distributed reverse proxy commonly used for quickly getting
-a public URL to a service running inside a private network, such as on your
-local laptop. The ngrok agent is usually deployed inside a private network and
-is used to communicate with the ngrok cloud service.
-
-This is the ngrok agent in library form, suitable for integrating directly into
-Rust applications. This allows you to quickly build ngrok into your application
-with no separate process to manage.
+[API Docs](https://ngrok.github.io/ngrok-rs/ngrok)
```

Comment:
Put the "unstable" back, or at least change it to "main".

There's a docs.rs badge up above that links to the ones auto-published to docs.rs for every version that we publish to crates.io, but these docs are sourced directly from the repo, and may have unreleased bits. Useful if someone is using the crate directly from git.